docs/streams.html

```diff
@@ -808,6 +808,12 @@
         <h2><a id="streams_upgrade_and_api" href="#streams_upgrade_and_api">Upgrade Guide and API Changes</a></h2>
 
         <p>
+        If you want to upgrade from 0.10.2.x to 0.11.0 you don't need to do any code changes as the API is fully backward compatible.
```

Comment:
@mjsax I was responding to @guozhangwang's question about `all Kafka modules`. Given that internal APIs are in the classpath and there is no warning when people use it (and from what I've seen people _do_ use them), there's a case to be made that it's worth mentioning it. One way to put it is `reality` versus `theory`. I'll leave it to you all to decide though.